src/core_plugins/kibana/public/discover/components/field_chooser/discover_field.js

```diff
@@ -14,9 +14,16 @@ app.directive('discoverField', function ($compile) {
     restrict: 'E',
     template: html,
     replace: true,
+    scope: {
+      field: '=',
+      onAddField: '=',
+      onAddFilter: '=',
+      onRemoveField: '=',
+      onShowDetails: '=',
+    },
     link: function ($scope, $elem) {
       let detailsElem;
-      let detailScope = $scope.$new();
+      let detailScope = null;
```

Comment:
no, just habit from languages where this could blow up :wink: thanks for pointing it out